jib-core/src/main/java/com/google/cloud/tools/jib/image/Image.java

```diff
@@ -100,8 +98,8 @@
      *     appear in the configuration json (e.g. "portNum/tcp")
      * @return this
      */
-    public Builder<T> setExposedPorts(Map<String, EmptyStruct> exposedPorts) {
-      this.exposedPorts = ImmutableSortedMap.copyOf(exposedPorts);
+    public Builder<T> setExposedPorts(List<String> exposedPorts) {
```

Comment:
I think we can actually change these to `ImmutableList` to avoid doing a `copyOf` on another `ImmutableList`, or just leave a TODO for now.